src/main/java/org/fundacion/jala/converter/controller/ExtractMetadataController.java

```diff
@@ -10,17 +10,12 @@
 
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
-import org.fundacion.jala.converter.service.ExtractMetadata;
+import org.fundacion.jala.converter.models.facade.ExtractFacade;
 import org.fundacion.jala.converter.service.FileStorageService;
-import org.fundacion.jala.converter.service.ObjectMetadata;
-import org.fundacion.jala.converter.service.metadata.TypeFileExport;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;

```

Comment:
You should only import what is necessary, not the entire library "*"

Reply:
solve